com.muensterimorbit.game.yaml

```diff
@@ -0,0 +1,17 @@
+id: com.muensterimorbit.game
```

Comment:
the last component of the id is the name of the application. "game" isn't acceptable.

Also `muensterimorbit.com` doesn't exist. Yet you acknowledged:

> The domain used for the application ID is self-assigned by the developer [https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository] and the application id guidelines [https://docs.flathub.org/docs/for-app-authors/requirements#application-id] are followed.